spec/controllers/catalog_controller_spec.rb

```diff
@@ -221,5 +236,23 @@
       post :update, params: { druid: druid, incoming_version: ver, incoming_size: size, storage_location: storage_location_param }
       expect(response.body).to include(druid_no_pre)
     end
+    context 'can take two forms of druid input' do
+      before do
+        post :create, params: { druid: druid, incoming_version: ver, incoming_size: size, storage_location: storage_location_param }
+      end
+      let(:druid) { 'druid:jj925bx9565' }
+      let(:druid_no_pre) { 'jj925bx9565' }
+
+      it 'druid:jj925bx9565' do
+        patch :update, params: { druid: druid, incoming_version: 5, incoming_size: size, storage_location: storage_location_param }
+        expect(response.body).to include(druid_no_pre)
```

Comment:
the `response.body` is equal to `"{\"druid\":\"jj925bx9565\",\"result_array\":[{\"actual_vers_gt_db_obj\":\"actual version (5) greater than PreservedCopy db version (3)\"}]}"`

Just thought i'm not testing for the result array so I could just test for the druid - happy to change it though

Reply:
Looking this over again, I actually don't have the same worry, since the expectation in on `druid_no_pre`, I was probably just overly concerned from the previous context. This one is ok by me.

Reply:
Testing for the exact body response makes the test pretty brittle when something trivial like a new key/value is added to the response.  Tests should try to avoid that when possible.

Reply:
are we expecting it NOT to include prefixed_druid  as well as  including  bare_druid?    You could write expectations to make this abundantly clear.